test/suites/ekuiper/streams_rules_test.go

```diff
@@ -58,10 +67,39 @@ func TestStreamsAndRules(t *testing.T) {
 	utils.TestDeviceVirtualReading(t)
 
 	t.Run("check rule_log", func(t *testing.T) {
-		utils.Exec(t, `edgex-ekuiper.kuiper-cli getstatus rule rule_log`)
+		//check logs for the record of expected log
+		start := time.Now()
+		time.Sleep(1 * time.Second)
+		logs := utils.SnapLogs(t, start, ekuiperSnap)
```

Comment:
Not sure what is the purpose of this. Sleep one second and query logs since a second ago?

Reply:
It is looking for the sink_log's result in the log. It worked locally. I am wondering if maybe the duration needs to be extended.